### PR TITLE
reference: Add documentation for static deltas

### PR DESCRIPTION
**Pull request by doanac**
Signed-off-by: Andy Doan <andy@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
this is a more formalized (with less commentary details) version of https://github.com/foundriesio/website/pull/28/files

File: source/reference-manual/ota/production-targets.rst [hunk 1]
```diff
@@ -37,6 +37,12 @@ production. A wave can be initialized with::
 
   fioctl wave init -k <path to offline-target-creds>.tgz v2.0-update 42 production
 
+.. note::
+
+   Operators will want to generate
+   :ref:`OSTree static deltas<ref-static-deltas>` before rolling
```
Comment (vanmaegima) [inline source/reference-manual/ota/production-targets.rst:43]: @doanac the checks have failed, I wonder if this file is missing

**Comment by doanac** [comment]
Docs for c2aa6ae57b16c081719bfb991f3c1b288795a52f are browesable at: https://ci.foundries.io/projects/fio-docs/builds/873/docs/artifacts/html/index.html

File: source/reference-manual/ota/static-deltas.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+In this example, Target #9 has passed CI and needs to be deployed to
+devices following the ``promoted`` tag. The operator can determine
+what static deltas are needed by running:
+
+.. prompt:: bash host:~$, auto
+
+   host:~$ fioctl targets static-deltas --dryrun --by-tag promoted 9
+   Dry run: Would generated static deltas for target versions:
+   7 -> 9
+   5 -> 9
```
Comment (vanmaegima) [inline source/reference-manual/ota/static-deltas.rst:48]: For some reason, this is what I'm seeing at the compiled page:
```
fioctl targets static-deltas --dryrun --by-tag promoted 9
ry run: Would generated static deltas for target versions:
 -> 9
 -> 9
```
The first character of each line is cropped, maybe try adding some leading whitespaces?

**Review by vanmaegima — APPROVED** [review]
Just a syntax comment, otherwise LGTM

File: source/reference-manual/ota/static-deltas.rst (hunk 2 above)
Comment (doanac) [reply source/reference-manual/ota/static-deltas.rst]: good catch. https://github.com/foundriesio/docs/pull/213/commits/a79593eb42887013f38e8cf72643ed7ed662abae

**Comment by doanac** [comment]
Docs for a79593eb42887013f38e8cf72643ed7ed662abae are browesable at: https://ci.foundries.io/projects/fio-docs/builds/876/docs/artifacts/html/index.html

File: source/reference-manual/ota/static-deltas.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+   host:~$ fioctl targets static-deltas --by-tag promoted 9
+
+Once the static deltas are in place, Target #9 can be re-tagged so
+that "promoted" devices will apply the update:
+
+.. prompt:: bash host:~$, auto
+
+   host:~$ fioctl targets tag --tags master,promoted --by-version 9
+
```
Comment (angolini) [inline source/reference-manual/ota/static-deltas.rst:70]: Maybe a finishing line reinforcing that the last command is enough to enable the devices to look for updates can help the reader to understand that it's really done.

**Review by angolini — APPROVED** [review]

File: source/reference-manual/ota/static-deltas.rst (hunk 3 above)
Comment (doanac) [reply source/reference-manual/ota/static-deltas.rst:70]: https://github.com/foundriesio/docs/pull/213/commits/c716a3955f4fb1909212b107e1bc3ed593d0eb72

**Comment by doanac** [comment]
Docs for c716a3955f4fb1909212b107e1bc3ed593d0eb72 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/884/docs/artifacts/html/index.html

**Review by mike-sul — APPROVED** [review]
I guess we will need to follow up with a doc on applying static deltas to (or in conjunction with)  the wave-based updates.

**Comment by doanac** [comment]
force push squash

**Comment by doanac** [comment]
Docs for 5f7f0a9b8d2e84c09d7b933fc27c3bbf7e659d17 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/887/docs/artifacts/html/index.html

**Pull request merged**